app/components/shared/card_component.rb

```diff
@@ -0,0 +1,55 @@
+class Shared::CardComponent < GovukComponent::Base
+  include ViewComponent::SlotableV2
+
+  renders_many :action_items, "ActionItemComponent"
+  renders_many :body_items, "BodyItemComponent"
+  renders_many :header_items, "HeaderItemComponent"
+
+  attr_accessor :html_attributes, :id
+
+  def initialize(id: nil, classes: [], html_attributes: {})
+    super(classes: classes, html_attributes: html_attributes)
+    @id = id
+  end
+
+  class HeaderItemComponent < ViewComponent::Base
+    attr_accessor :label, :value
+
+    def initialize(value:, label: nil)
+      @label = label
+      @value = value
+    end
+
+    def call
+      (@label ? content_tag(:span, "#{@label}: ", { class: "govuk-!-font-weight-bold" }) : "") + @value
+    end
+  end
+
+  class BodyItemComponent < ViewComponent::Base
+    attr_accessor :label, :value
+
+    def initialize(value:, label: nil)
+      @label = label
+      @value = value
+    end
+
+    def call
+      (@label ? content_tag(:span, "#{@label}: ", { class: "govuk-!-font-weight-bold" }) : "") + @value
```

Comment:
if you have `attr_accessor :label, :value` you do not need the @ for @label and @value

Reply:
i knew this